### ISSUE TITLE
Tests fail without "." in @INC

### ISSUE THREAD

**Issue by eserte** [hitode909/APISchema#24]
If perl has no "." in `@INC` (which is the case for newer perls), then the test suite fails:
```
Can't locate t/test.pm in @INC (you may need to install the t::test module) (@INC contains: /usr/home/cpansand/.cpan/build/2017111709/APISchema-1.28-6/blib/arch /usr/home/cpansand/.cpan/build/2017111709/APISchema-1.28-6/blib/lib /usr/perl5.26.1p/lib/site_perl/5.26.1/amd64-freebsd /usr/perl5.26.1p/lib/site_perl/5.26.1 /usr/perl5.26.1p/lib/5.26.1/amd64-freebsd /usr/perl5.26.1p/lib/5.26.1) at t/APISchema-DSL.t line 2.
BEGIN failed--compilation aborted at t/APISchema-DSL.t line 2.
t/APISchema-DSL.t ................................. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
```

The solution is usually to add something like `use lib "."`.

**Comment by hitode909** [comment]
Thank you! I'll check it out.

**Comment by hitode909** [comment]
Thank you for reporting. I've fixed this issue in #25 and I'll release new version soon.
